### DISCUSSION TITLE
Ort-server deployment in k8s

### DISCUSSION THREAD

**Discussion by jacek-kupczyk-wenovate** [eclipse-apoapsis/ort-server#696] [Q&A] [answered]
Hi Team,

Following up on our previous introduction meeting about Ort-server, we were able to successfully run the latest version using Docker Compose. However, we noticed that the Helm chart is not up to date with this latest version.
Could you please let us know when you are planning to publish the updated Helm chart in your Git repository?
Thanks for your assistance!

Best regards,
Jacek Kupczyk from CARIAD

**Comment by sschuberth** [comment]
> However, we noticed that the Helm chart is not up to date with this latest version.

@haikoschol is this something you're also working on?

**Reply by haikoschol** [reply]
No, since we use kustomize instead of Helm. It should not be too much work to produce a basic chart from our manifests, depending on how flexible it needs to be. I might have some time to work on that in a week or two.

**Comment by mnonnenmacher** [comment]
In August I want to start the process to prepare the first official release of the project. As part of this also the documentation will be improved and hopefully we will also be able to provide an updated Helm chart. So realistically, unless someone else can contribute, don't expect it before September.

**Comment by jacek-kupczyk-wenovate** [comment]
@mnonnenmacher @sschuberth Thank you for the quick response. I will try to update the Helm chart on my own and get back to you with the results next week.

**Reply by sschuberth** [reply]
Any update @jacek-kupczyk-wenovate?

**Reply by mnonnenmacher** [reply]
I am currently working on deploying the ORT Server to an Azure environment. I am using a custom Helm chart in a test repository for that, but when it is done I will update the Helm chart in this repository.

**Comment by mnonnenmacher** [comment, marked as answer]
A Helm chart is now published in a separate repository:
https://github.com/eclipse-apoapsis/helm-charts
